pkgs/by-name/fr/freediameter/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  pkg-config,
+  flex,
+  bison,
+  lksctp-tools,
+  libidn2,
+  libgcrypt,
+  gnutls,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "freediameter";
+  version = "1.5.0-unstable-2025-03-16";
+
+  src = fetchFromGitHub {
+    owner = "freeDiameter";
+    repo = "freeDiameter";
+    rev = "8e525acdfd439995f3e8e26d5a802fc4ad95d24c";
+    hash = "sha256-ai2R8scP++tdPh303RAl0qdIpehzFoyykAuAyl2w3MA=";
+  };
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    flex
+    bison
+    lksctp-tools
+    libidn2
+    libgcrypt
+    gnutls
+  ];
+
+  configurePhase = ''
+    mkdir -p build
+    cd build
+    cmake ..
+  '';
+
+  buildPhase = ''
+    cmake --build .
+  '';
+
+  installPhase = ''
+    cmake --install . --prefix $out
+  '';
+
+  meta = {
+    description = "Implementation of the Diameter Protocol for Authentication, Authorization, and Accounting";
+    homepage = "https://github.com/freeDiameter/freeDiameter";
+    license = lib.licenses.bsd3;
+    platforms = lib.platforms.linux ++ lib.platforms.freebsd;
```

Comment:
This should also build [on darwin](https://github.com/freeDiameter/freeDiameter/blob/master/INSTALL.OSX), no? If so, we may have this as `lib.platforms.unix`